src/ballot-forms.php

```diff
@@ -1,36 +1,33 @@
 <?php
 require_once 'includes/classes/db-connector.php';
 require_once 'includes/session-handler.php';
-require_once 'includes/classes/session-manager.php';
 
 
 if(isset($_SESSION['voter_id'])) {
```

Comment:
Your page is prone to directory listing. You're checking if a user's session is set, but you perhaps forgot to also check his role. I don't if it's just on my copy, but as the video shows, the footer also looks distorted. See the video below.


https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/162773681/01808326-e647-4bfa-92a6-deff3fe90f67